app/lib/dttp/code_sets/institutions.rb

```diff
@@ -0,0 +1,198 @@
+# frozen_string_literal: true
+
+module Dttp
+  module CodeSets
+    module Institutions
+      MAPPING = {
+        "Aberystwyth University" => { entity_id: "443e2cff-6f42-e811-80ff-3863bb3640b8" },
+        "Anglia Ruskin University" => { entity_id: "" },
```

Comment:
Are all of these supposed to have empty `entity_id`-ses?

Reply:
Ben's working on the mapping as we have some duplicates